### ISSUE TITLE
Best practice for adding new data to an OTU. Check mark or combination one?

### ISSUE THREAD

**Issue by mabecabrera** [SpeciesFileGroup/taxonworks#3502]
### Feature or enhancement

The migration of the OSF database is linking all information about an OTU (of a species that is currently used in a different combination from the genus in which it was originally described) to the protonym one (the one with the checkmark). The question is: In the future, should we enter new information attached to the combination? Or we continue to add it to the original protonym?

What would be the consequences?

Based on this text in the Taxon Works manual:
`Enter information on an existing taxon
Before you do this, we recommend you add the Browse OTU card to your favorites for easy quick access.
Navigate to the taxon (OTU) using the Browse OTU card in tasks
HINT: There is at least one important caveat. If you are capturing information for a species that is currently used in a different combination from the genus in which it was originally described, you may get two OTUs for the species name when you search for it in Browse OTU.
One will be the OTU attached to the original protonym (indicated by a check mark next to the name)
One will be the OTU attached to the combination (indicated by a c next to the name).
Use the OTU attached to the combination, at least for now. This is an area of the database that is still being worked on, but this should facilitate downstream display of the new information.
`

### Location

task New images

### Screenshot, napkin sketch of interface, or conceptual description

![image](https://github.com/SpeciesFileGroup/taxonworks/assets/63611746/8d9d8c1f-d951-4c47-ba9e-b91fb75f8ca7)

### Your role

Curator

**Comment by mabecabrera** [comment]
So this was a practice for UCD. We missed the note in the intro.
So each project should manage this kind of decisions.

**Comment by mabecabrera** [comment]
Also I have a question... Sometimes it is still confusing for me. When I have a Taxon Name that happens to be an original misspeling. This Taxon name original misspelled has its OTU and the well spelled one ALSO has its OTU... but technically they hare exactly the same OTU... I should add any data to the correctly spelled right? That is confusing for me because we have this OTU concept but I feel sometimes it serves a technical matter more than a biological one and makes it hard to understand this biological unit called OTU.

**Comment by mjy** [comment]
Agreed, there is something that feels not as smooth as it might be and your feeling for technical vs. biological is right on. In SFs there was nomenclators that sat between the data, i.e. similar mechanisms, but with maybe different meaning.  

Here we treat OTU somewhat nebulously, the biological unit we do work with.  If you think of OTUs as some useful biological package of data that can be combined at different levels, then it might help.  Many packages of biological concept, all related, combine to form, say, a species circumscription.  

We presently `coordinate` OTUs by proxy of the nomenclature, this mostly works, but there is a second approach as well, the OTURelationship.  Strictly speaking the latter is the better way to combine things, then only  applying names after you figure out where the type specimen is, but in practice few people dissect the data at that level (though some are very interested to).  If we use OTURelationships then we could coordinate by that proxy, which may or may not equal the nomenclature based one (in fact it likely doesn't when you look close).

So, since we have potentially many ways to group data, and re-group it the OTU becomes a useful anchor to hang data off of, yes, it perpetuates where we see similar names, but it also gives us great power to refactor what we have asserted.  In many cases the OTU is conceptually the same as another, but in many other cases the authors use might be subtly different.  So we jump through some hoops (i.e. seeing Combination based OTUs in our lists) to 1) keep historical context for 2) things that may/not be exaclty the same concept and 3) coordinate those data to present a unified view.  

Probably not as satisifying an answer as you're looking for, it's a tricky one, and there might be better models out there, but its what we have. Maybe this helps.
